### PR TITLE
[EI-92] Cache overview documentation

### PR DESCRIPTION
**Pull request by jspellman814**
Additions to #6861 to close out [EI-92](https://getpantheon.atlassian.net/browse/EI-92)

### CODE REVIEW

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | [Live](https://pantheon.io/docs) | [pr-7186](https://pr-7186--pantheon-docs.my.pantheonfrontend.website/docs) | Difference |
| --- | --- | --- | --- |
| Performance Score | `62` | `61` | `-1` |
| Best Practice Score | `79` | `79` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-7186` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/34830/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/34830/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)


[Preview Environment](https://pr-7186--pantheon-docs.my.pantheonfrontend.website/docs)

File: source/content/guides/edge-integrations/02-configuration-overview.md [hunk 1]
```diff
@@ -28,9 +30,12 @@ When you personalize with AGCDN, the content variant generated by the set of con
 
 Your CMS processes the original content, makes a determination, and displays a first-time rendering of the content. Once the content is rendered for the first time, it’s being saved in the CDN cache layer alongside the conditional header; your CMS isn’t involved until the cache expires, content is updated, or the conditional variant doesn’t exist in the cache layer.
 
-## Configuration
+## Cache Overview
 
+A `Vary` response header tells caches that a particular header (or headers) from the request should be used to construct a secondary cache key that must match in order to use the cached response to answer subsequent requests.
```
Comment (jazzsequence) [inline source/content/guides/edge-integrations/02-configuration-overview.md:35]: This is a specific behavior of our (Fastly) cache, not necessarily applicable to _all_ caching. I think this should be more specific to indicate that this is a feature of AGCDN, specifically.

```suggestion
A `Vary` response header tells Pantheon's AGCDN cache that a particular header (or headers) from the request should be used to construct a secondary cache key that must match in order to use the cached response to answer subsequent requests.
```

File: source/content/guides/edge-integrations/02-configuration-overview.md [hunk 2]
```diff
@@ -28,9 +30,12 @@ When you personalize with AGCDN, the content variant generated by the set of con
...
 
+For example, when the Vary header is set for Interest, the a server produces versions of the same webpage with different Interests and serves them to visitor's browsers based on their locally defined interests.
```
Comment (jazzsequence) [inline source/content/guides/edge-integrations/02-configuration-overview.md:37]: Hyphenate "locally-defined" to show possession.

```suggestion
For example, when the Vary header is set for Interest, the a server produces versions of the same webpage with different Interests and serves them to visitor's browsers based on their locally-defined interests.
```

**Review by jazzsequence — CHANGES_REQUESTED** [review]
A couple minor tweaks, otherwise lgtm

File: source/content/guides/edge-integrations/02-configuration-overview.md (hunk 1 above)
Comment (jspellman814) [reply source/content/guides/edge-integrations/02-configuration-overview.md]: Addressed in [8b7ae81](https://github.com/pantheon-systems/documentation/pull/7186/commits/8b7ae81a4e4d560c9af7cf5e14e072036385d502).

File: source/content/guides/edge-integrations/02-configuration-overview.md (hunk 2 above)
Comment (jspellman814) [reply source/content/guides/edge-integrations/02-configuration-overview.md]: Addressed in [8b7ae81](https://github.com/pantheon-systems/documentation/pull/7186/commits/8b7ae81a4e4d560c9af7cf5e14e072036385d502).

**Review by jazzsequence — APPROVED** [review]
lgtm

**Review by michellecolon-pantheon — APPROVED** [review]
LGTM

**Pull request merged**